### PR TITLE
Features workflows

### PR DESCRIPTION
**Pull request by rebel216**
Added features-workflow branch .Used it to commit changes such as added HTML , CSS ,Linter files.
Goal was to check Linters functioning and learn about the process on how to use linters in Github and local Dev-environment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,6 @@
-# Hello-Microverse
```
Comment (OLIPLICHE) [inline README.md:1]: - [x] Kindly add a descriptive README file to your project. Please use the template provided by Microverse to customize your Readme. Here is the [Link](https://github.com/microverseinc/readme-template/blob/master/README.md) to the Template

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi, @rebel216, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter result is green (✅) 👏 
- Commit messages are meaningful 👏 

## Required Changes ♻️

- [ ] Please add a descriptive title in your pull Request telling you what changes are inside.
- [ ] Please your Pull Request need have a summary describing introduced changes in more detail

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (rebel216) [reply README.md:1]: hello. How to add description to pull request without creating new pull request.

**Comment by rebel216** [comment]
Added features-workflow branch .Used it to commit changes such as added HTML , CSS ,Linter files.
Goal was to check Linters functioning and learn about the process on how to use linters in Github and local Dev-environment.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Hello</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <script src="index.js"></script>
+    <h1>Hello Microverse!</h1>
+  </body>
```
Comment (m05e5) [inline index.html:13]: - _(optional)_ Please as part of the requirement, its better to put your "< h1 >" tag in a "< header >" tag for your code to be more semantic and for SEO

**Review by m05e5 — APPROVED** [review]
# Hi @rebel216 , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[Highly Recommended]
- _( optional )_ Please as part of the requirement, its better to put your "< h1 >" tag in a "< header >" tag for your code to be more semantic and for SEO

- _(optional)_ Your Pull Request should have a descriptive title telling you what changes are inside(you can edit it at the top of the PR).


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (rebel216) [reply index.html:13]: Thanks I will :)

**Pull request merged**